src/main/java/UseCases/FridgeUseCase.java

```diff
@@ -0,0 +1,50 @@
+package UseCases;
+
+import Commands.Command;
+import Commands.Implementations.CommandImpl;
+import Storages.Implementations.IngredientStorageImpl;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+
+public class FridgeUseCase implements UseCase {
+    IngredientStorageImpl ingredientStorage;
+    IngredientStorageImpl fridge;
+
+    /**
+     * Constructor
+     */
+    public FridgeUseCase(IngredientStorageImpl fridge, IngredientStorageImpl ingredientStorage) {
+        this.ingredientStorage = ingredientStorage;
+        this.fridge = fridge;
+
+    }
+
+    @Override
+    public IngredientStorageResponseImpl run(CommandImpl command) {
+        // checks if the user is using addToFridge command (non-empty)
+        if (!Objects.equals(command.get("addtofridge"), "") && command.containsKey("addtofridge")) {
```

Comment:
This is redudant, `get` will return null if there is no key. No need to check the existence of the key.